pkgs/gnolang/gnomod/file.go

```diff
@@ -33,34 +35,77 @@ func (f *File) Validate() error {
 // FetchDeps fetches and writes gno.mod packages
 // in GOPATH/pkg/gnomod/
 func (f *File) FetchDeps(remote string) error {
+	var indirect string
```

Comment:
I suggest you move this in the for loop (smaller scope) for clarity.

Reply:
Done: b57b921465ac64bea8bdaf9bd0b5ea0369865b38